http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberSpec.scala

```diff
@@ -566,7 +563,7 @@ class LeoPubsubMessageSubscriberSpec
     res.unsafeRunSync()
   }
 
-  it should "not handle DeleteDiskMessage when disk is not in Deleting status" in isolatedDbTest {
+  it should "handle DeleteDiskMessage when disk is not in Deleting status" in isolatedDbTest {
```

Comment:
if we indeed don't need to check if disk is `Deleting` then we should update the test name above accordingly

Reply:
Thats what I intended here with the wording change. Is it not clear?

Reply:
Having `Deleting` status is no longer a special case so I'd think we not mention it in the test name, i.e.  "handle DeleteDiskMessage". What am I missing?